### ISSUE TITLE
ui5-panel component: It violates i11y rule when we use it as fixed.

### ISSUE THREAD

**Issue by lateefsofi** [SAP/ui5-webcomponents#1753]
**Description**
ui5-panel violates i11y rule when it is used with fixed property. We run automated tests to check the violations. We use axe for a11y testing. 
Looks there is an svg element in the dom which has violations.
**To reproduce**
1. Add follwoing code to your code base(We use it in ReactJs code base)
`<ui5-panel data-testid="related-topics-container" fixed accessible-role="Complementary" header-text={i18n.getText("label.RelatedTopics")} class={styles.topicsContainer}>
//Component contents here
</ui5-panel>`
2. Write unit tests:
Get the reference(example, container) of the container of the component where this component is used.
then `expect(container).toHaveNoA11yViolations()`
3. ...were 

**Isolated example**
Please provide an isolated example if possibile, by forking [CodeSandbox](https://codesandbox.io/s/71r1x5o51q?fontsize=14&module=%2Findex.html)

**Expected behavior**
There shouldn't be any a11y violation.

**Screenshots**
![image](https://user-images.githubusercontent.com/18447407/84107723-0a8d1800-aa3c-11ea-8a7d-e0742535f275.png)

**Context**
- UI5 Web Components version: "^1.0.0-rc.7"
- OS/Platform: Windows
- Browser *(if relevant)*: NA
- Other information: "@testing-library/react" used for testing

**Affected components** *(if known)* NA

**Log output / Any errors** in the console
```
![image](https://user-images.githubusercontent.com/18447407/84107671-e92c2c00-aa3b-11ea-898d-30286a87ee3f.png)

```

**Comment by ilhan007** [comment]
Hello @lateefsofi 

 We did a lot of changes since 1.0.0-rc.7, could you test with the "next" tag (version 0.0.0-d849bec2c
) to be sure you test against the latest development

<img width="726" alt="Screenshot 2020-06-09 at 8 36 04 AM" src="https://user-images.githubusercontent.com/15702139/84109871-5a63e300-aa2c-11ea-9a04-28e763c360d1.png">

**Comment by vladitasev** [comment]
Hello,

I can confirm that in the latest version the svg has role="presentation" and not "img".
The change in particular is: https://github.com/SAP/ui5-webcomponents/pull/1652

![image](https://user-images.githubusercontent.com/15844574/84112934-52a73d00-aa32-11ea-9b13-c2747d831c9b.png)

Regards,
Vladi

**Comment by vladitasev** [comment]
Reopening since there is still a problem when "fixed" is set
